news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -12,115 +12,118 @@ tags:
 ---
 
 
-The journey of the release of PrestaShop 1.7.7.0 release has started. This moment is a nice opportunity to explain fully the PrestaShop minor releases lifecycle.
+The journey of the release of PrestaShop 1.7.7.0 has started. This moment is a nice opportunity to explain PrestaShop's minor release lifecycle.
 
-# The journey begins: Kanban and scope
+## The journey begins: Kanban and scope
 
-Each minor version is defined by a feature scope, and this scope is defined as a number of github issues that we put into a [Kanban](https://help.github.com/en/github/managing-your-work-on-github/about-project-boards) (see [1.7.7 Kanban](https://github.com/PrestaShop/PrestaShop/projects/7)). How these issues are picked or sorted is the responsibility of the Product Team which spends a lot of time gathering feedback about all PrestaShop users to make sure the next minor version addresses the most important matters and needs.
+Each minor version is defined by a feature scope, which is defined by a number of GitHub issues that we put in a version [Kanban](https://help.github.com/en/github/managing-your-work-on-github/about-project-boards) (see [1.7.7 Kanban](https://github.com/PrestaShop/PrestaShop/projects/7)). How these issues are picked or sorted is the responsibility of the Product Team, which spends a lot of time gathering feedback from the PrestaShop community to make sure the next minor version addresses the most important needs.
```

Comment:
Minor improvement to avoid repeating "defined"
```suggestion
Each minor version is defined by a feature scope, that is, a number of GitHub issues that we put in a version [Kanban](https://help.github.com/en/github/managing-your-work-on-github/about-project-boards) (see [1.7.7 Kanban](https://github.com/PrestaShop/PrestaShop/projects/7)). How these issues are picked or sorted is the responsibility of the Product Team, who spends a lot of time gathering feedback from the PrestaShop community to make sure the next minor version addresses the most important needs.
```